weaviate/batch/crud_batch.py

```diff
@@ -805,11 +805,19 @@ def _readd_objects_after_timeout(
         new_batch = ObjectsBatchRequest()
         for obj in batch_request.get_request_body()["objects"]:
             class_name = obj["class"]
+            tenant = obj.get("tenant", None)
             uuid = obj["id"]
-            response_head = self._connection.head(
-                path="/objects/" + class_name + "/" + uuid,
-            )
-
+            
+            if tenant is None:
+                response_head = self._connection.head(
+                    path="/objects/" + class_name + "/" + uuid,
+                )
+            else:
+                response_head = self._connection.head(
+                    path="/objects/" + class_name + "/" + uuid,
+                    params={"tenant": tenant},
+                )
```

Comment:
could you just make params conditional?

eg
```
tenant = obj.get("tenant", None)
params = {"tenant": tenant} id tenant is not None else None
response_head = self._connection.head(
     path="/objects/" + class_name + "/" + uuid,
     params=params,
)
```

and then the same for the get request below (just reuse params)